ios/app/app.xcodeproj/project.pbxproj

```diff
@@ -319,16 +336,16 @@
 				TargetAttributes = {
 					0BEA5C241F7B8F73000D0AB4 = {
 						CreatedOnToolsVersion = 9.0;
-						DevelopmentTeam = FC967L3QRG;
+						DevelopmentTeam = S76GCAG929;
```

Comment:
I guess you needd to revert this, or remove the option entirely?

Reply:
Thanks, correct. This was an artefacts from me trying to make sure everything works.